src/webapp/src/client/resources/en_US/kudos.json

```diff
@@ -20,7 +20,7 @@
   "kudosType": "Type",
   "multiplier": "Multiplier",
   "points": "Points",
-  "kudos.amountOfKudos": "Amount of Kudos",
```

Comment:
Shouldn't it be changed in other language files as well?

Reply:
It already existed in en_US language
![image](https://user-images.githubusercontent.com/51354779/67271928-31ee9f00-f4c4-11e9-94ae-f8971c54b4b5.png)
So just fixed the naming. (It had no references)

Other file already had this translation:
![image](https://user-images.githubusercontent.com/51354779/67272087-87c34700-f4c4-11e9-9aa5-edd9bbe9012b.png)